update-plex.sh

```diff
@@ -108,6 +108,20 @@ EOF
   fi
 }
 
+function retrieve_dsm_version() {
+  header "Retrieving DSM version"
+  dsm_major_version=$(awk -F "=" '/majorversion/ {print $2}' /etc.defaults/VERSION | tr -d '"')
+  if [[ "7" -eq "$dsm_major_version" ]]; then
+    json_dsm_pattern="Synology (DSM 7)"
+    pms_package_name="PlexMediaServer"
+    echo "Found DSM version 7"
+  else
+    json_dsm_pattern="Synology"
+    pms_package_name="Plex\ Media\ Server"
```

Comment:
Hi, I tested this on DSM6 and found the following error, this should be:
`pms_package_name="Plex Media Server"`
(so no backslash on the spaces), when fixing this it solved my problem with updates no longer working on DSM6 (I was on release v1.0.3).

Reply:
My guess was that the command `synopkg version Plex Media Server` would not work without the backslashes, can you try it out on your NAS ? If it works I will update the PR, thanks for the review.

Do you mind providing the complete output of the script too ?

Reply:
Sure, I have tried this on my actual DS918 (with 6.2.4-25556), and this is the result with the backslashes:
```
Task: Update Plex
Start time: Tue, 06 Jul 2021 11:42:07 GMT
Stop time: Tue, 06 Jul 2021 11:42:08 GMT
Current status: 1 (Interrupted)
Standard output/error:
Checking for a Plex Media Server update...

[ Retrieving DSM version ]
Found DSM version <6

[ Enabling Plex Pass releases ]
Found Plex Token

[ Retrieving version data ]
Available version: 1.23.4.4775-b7e0c30f2

Done, with errors!
```
When I edited the script and removed the slashes, everything worked. 

You should not both enclose a variable in quotes, and escape the spaces since the backlashes will then be preserved in the variable, and you will get an incorrect string (with backslashes). If you look in the code where the variable is used it is always quoted (which is a good practice btw), like: `installed_version=$(synopkg version "$pms_package_name")`

Reply:
Oh ok my bad, thank you for the explanation, I've updated the PR accordingly.

Reply:
No worries, thanks for the good PR :-)